scripts/generic-ci-tests.sh

```diff
@@ -77,8 +77,6 @@ case "$TEST_SUITE" in
 
         mkdir -p reports
 
-        echo "Finding fixme's and storing report..."
-        run_paver_quality find_fixme || { EXIT=1; }
```

Comment:
This step doesn't do anything because there is no pylint report to read the fixme's from. It has been like this ever since pylint was moved into its own github action, ~2021.